pkg/download/classic/download.go

```diff
@@ -82,6 +84,22 @@ func Download(client client.ConfigClient, projectName string, apisToDownload api
 	return results, nil
 }
 
+func checkAndRemoveValuesWithDuplicateIDs(api api.API, originalValues values) values {
+	seenIDs := make(map[string]struct{}, len(originalValues))
+	filteredValues := make(values, 0, len(originalValues))
+
+	for _, v := range originalValues {
+		if _, alreadySeen := seenIDs[v.value.Id]; alreadySeen {
+			log.Warn("Received multiple '%s' configs with the same ID '%s'; skipping duplicate named '%s'", api.ID, v.value.Id, v.value.Name)
```

Comment:
Is it a good idea to include only one configuration and reject all others? 
In this way, the complete Monaco process will end successfully, and a user can interpret that as meaning that all configurations have been downloaded.

Reply:
As discussed, it's ok as is here